src/components/ObjektDisplay.vue

```diff
@@ -0,0 +1,48 @@
+<script setup lang="ts">
+    const objektsQuery = `
+        query MyQuery 
+            {collections(limit: 18){
+                front
+                id
+            }
+        }
+        `;
+        
+    const response = await fetch("https://cosmo-api.gillespie.eu/graphql", {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json"
+        },
+        body: JSON.stringify({
+            query: objektsQuery,
+        })
+    });
+
+    const parsedData = await response.json();
+    const imageList = parsedData.data.collections
+
+
+
+    console.log(imageList);
+</script>
+
+<template>
+    <div id="title">
+        <h2>10 first objekts query</h2>
+    </div>
+    
+    <div class="container">
+        <img v-for="singleImage in imageList" v-bind:src="singleImage.front" v-bind:alt="singleImage.id" width="100%"/>
```

Comment:
Instead of doing `v-bind:src` you can do just `:src`.

This is a short hand and means the same. You will see this much more frequently as well.